test/test_suite_main_testcases.hpp

```diff
@@ -766,6 +766,35 @@ PRINTF_TEST_CASE(floating_point_specifiers_precision_and_flags)
 {
   char buffer[base_buffer_size];
 #if PRINTF_SUPPORT_DECIMAL_SPECIFIERS
+  /* %f for float that prompt to double */
```

Comment:
1. Did you mean to say "promote" rather than "prompt?
2. These checks are not useful, since floats are promoted to doubles when passed to a variadic function like printf, so that this:
    ```
    PRINTING_CHECK("3.1415", ==, sprintf_, buffer, "%.4f", 3.1415354f)
    ```
    is the same as this:
    ```
    PRINTING_CHECK("3.1415", ==, sprintf_, buffer, "%.4f", 3.1415354)
    ```

Reply:
Yeap, `promote`, this is a test to clarify `PRINTING_CHECK("3.1415", ==, sprintf_, buffer, "%.4f", 3.1415354f)` is the same with `PRINTING_CHECK("3.1415", ==, sprintf_, buffer, "%.4f", 3.1415354)`